doc/reference/networking/can_api.rst

```diff
@@ -0,0 +1,282 @@
+.. _can_interface:
+
+Controller Area Network (CAN)
+#############################
+
+.. contents::
+    :local:
+    :depth: 2
+
+Overview
+********
+
+Controller Area Network is a two-wire serial bus specified by the
+Bosch CAN Specification, Bosch CAN with Flexible Data-Rate specification and the
+ISO 11898-1:2003 standard.
+CAN is mostly known for its application in the automotive domain. However, it
+is also used in home and industrial automation and other products.
+
+A CAN transceiver is an external device that converts the logic level signals
+from the CAN controller to the bus-levels. The bus lines are called
+CAN High (CAN H) and CAN Low (CAN L).
+The transmit wire from the controller to the transceiver is called CAN TX,
+and the receive wire is called CAN RX.
+These wires use the logic levels whereas the bus-level is interpreted
+differentially between CAN H and CAN L.
+The bus can be either in the recessive (logical one) or dominant (logical zero)
+state. The recessive state is when both lines, CAN H and CAN L, at roughly at
+the same voltage level. This state is also the idle state.
+To write a dominant bit to the bus, open-drain transistors tie CAN H to Vdd
+and CAN L to ground.
+The first and last node use a 120-ohm resistor between CAN H and CAN L to
+terminate the bus. The dominant state always overrides the recessive state.
+This structure is called a wired-AND.
+
+.. warning::
+
+   CAN controllers can only initialize when the bus is in the idle (recessive)
+   state for a specific amount of time. Therefore you have to make sure that
```

Comment:
How long is a "specific amount of time" and "for a short time"?  Are these defined in the CAN spec?

Reply:
I found some notes in the data-sheets. Need to check the standard for that,

Reply:
It is 11 bit. Updated the docu accordingly